Master qa 51497 1

dellawang opened the pull request.

jbalsas committed:
LPS-95863 Consolidates API as Session instead to avoid confusion

mateomustapic committed:
LPS-95863 Modernize Store utilities

mateomustapic committed:
LPS-95863 Tests for Store utilities

aline-doleron committed:
LPS-94150 Completely display footer of Delete field modal by overwriting justify-content css property of Clay to be compatible with IE11 and other browsers

Commit:
LPS-99791 Re-index User when indexable fields are updated

aline-doleron committed:
LPS-95525 Overwrite Clay's toogle-switch-label max width to none to have the same visual result on IE11 and other browsers

Commit:
LPS-99791 Regen

kresimir-coko committed:
LPS-98278 dynamic-data-mapping - Update PortletURL usages

michaelhashimoto committed:
LRCI-511 Add secret volumes & mounts to oracle pod configurations

michaelhashimoto committed:
LRCI-511 Add support for copying files into oracle pods

drewbrokke committed:
LPS-99964 frontend-js-web - (liferay.js) SF only - makes setting the http method functions on Liferay.Service more explicit so they are easier to search for

pyoo47 committed:
LRCI-511 Simplify

pyoo47 committed:
LRCI-511 Sort

jbalsas committed:
LPS-95863 Updates Store tests

yichenroy committed:
LRCI-566 Allow specifying sender when creating central merge pull request

Songyuewen committed:
LRQA-51487 Update Calendar#CURRENT_SITE_CALENDARS_CALENDAR_MENULIST_ARROW locator

Songyuewen committed:
LRQA-51485 Update BlogsEntry#FLAG_MODAL locator

Songyuewen committed:
LRQA-51485 Update ReportContent#REASON_SELECT locator

dellawang committed:
LRQA-51497 add if to check user login or not

dellawang committed:
check user login or not

dellawang committed:
only ci

dellawang committed:
tab

dellawang committed:
sf

Commit:
LPS-100189 Replaces legacy mixins in blogs-web

Commit:
LPS-100191 Replaces legacy mixins in contacts-web

Commit:
LPS-100194 Replaces legacy mixins in dynamic-data-mapping-web

Commit:
LPS-100195 Replaces legacy mixins in fragment-web

Commit:
LPS-100201 Replaces legacy mixins in journal-web

Commit:
LPS-100203 Replaces legacy mixins in layout-admin-web

Commit:
LPS-100204 Replaces legacy mixins in layout-content-page-editor-web

Commit:
LPS-100209 Replaces legacy mixins in product-navigation-control-menu-theme-contributor

Commit:
LPS-100211 Replaces legacy mixins in product-navigation-product-menu-web

Commit:
LPS-100213 Replaces legacy mixins site-navigation-admin-web

Commit:
LPS-100189 Update the imported mixins libraries

kresimir-coko committed:
LPS-99699 app-builder - Update fetch usages

Commit:
LPS-100191 Update the imported mixins libraries

Commit:
LPS-100194 Update the imported mixins libraries

Commit:
LPS-100201 Update the imported mixins libraries

Commit:
LPS-100203 Update the imported mixins libraries

Commit:
LPS-100204 Update the imported mixins libraries

Commit:
LPS-100209 Update the imported mixins libraries

Commit:
LPS-100211 Update the imported mixins libraries

Commit:
LPS-100213 Update the imported mixins libraries

Commit:
LPS-100195 Update the imported mixins libraries

jbalsas committed:
LPS-95863 Format module's API definition

knchau committed:
LPS-100154 Change elementId to a unique value to avoid replacing existing element data with the same element name

yichenroy committed:
LRCI-566 Modify create pull request method to specify sender

holatuwol committed:
LPS-100274 Add a new method to avoid duplicate logic to check for whether to suppress SocketException

holatuwol committed:
LPS-100274 Apply to methods that use NIO (StreamUtil, FileChannel)

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

dellawang commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-4-1.liferay.com/job/test-portal-source-format/851/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-4-1.liferay.com//userContent/jobs/test-portal-source-format/builds/851/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-22.liferay.com/job/test-portal-source-format/3244/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-22.liferay.com//userContent/jobs/test-portal-source-format/builds/3244/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 2 minutes 31 seconds 652 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/44069c50e5236df066f705e5d88b04559f7e5883">44069c50e5236df066f705e5d88b04559f7e5883</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/dellawang/liferay-portal/tree/master-qa-51497-1">master-qa-51497-1</a><br/>Branch GIT ID: <a href="https://github.com/dellawang/liferay-portal/commit/87273ca9b764047f16e541bfa9f308ba391bd797">87273ca9b764047f16e541bfa9f308ba391bd797</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-22.liferay.com/job/test-portal-source-format/3244/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-22.liferay.com/userContent/jobs/test-portal-source-format/builds/3244/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 2 minutes 41 seconds 977 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/44069c50e5236df066f705e5d88b04559f7e5883">44069c50e5236df066f705e5d88b04559f7e5883</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/dellawang/liferay-portal/tree/master-qa-51497-1">master-qa-51497-1</a><br/>Branch GIT ID: <a href="https://github.com/dellawang/liferay-portal/commit/87273ca9b764047f16e541bfa9f308ba391bd797">87273ca9b764047f16e541bfa9f308ba391bd797</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-4-1.liferay.com/job/test-portal-source-format/851/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-4-1.liferay.com/userContent/jobs/test-portal-source-format/builds/851/jenkins-report.html">here</a>.</h5></details></html>

jbalsas committed:
LPS-99699 Adds missing frontend-js-web dependency

kresimir-coko committed:
LPS-98278 dynamic-data-mapping - SF and add .toString()

jbalsas committed:
LPS-100189 SF

jbalsas committed:
LPS-100191 SF

jbalsas committed:
LPS-100194 SF

jbalsas committed:
LPS-100195 SF

Commit:
LPS-100059 Add localizable attribute

Commit:
LPS-100059 Expose "system" property of field types

Commit:
LPS-100059 Expose method to retrieve field types

Commit:
LPS-100059 Use 'componentId' to identify container

Commit:
LPS-100059 buildTagLib

Commit:
LPS-100059 Add FieldType component

Commit:
LPS-100059 Add UpperToolbar component

Commit:
LPS-100059 Improve Sidebar animations

Commit:
LPS-100059 Add hook to facilitate animating content with a Sidebar on the side

Commit:
LPS-100059 Removes extra padding

Commit:
LPS-100059 Allow Table to support click events on items

Commit:
LPS-100059 Use UpperToolbar

Commit:
LPS-100059 Allow editting Form Views

Commit:
LPS-95954 Add dependencies

Commit:
LPS-95954 Add lang keys

Commit:
LPS-100342 Build ServiceContext before ThemeDisplay otherwise the previously existing ServiceContext referres to a request originating before friendlyURL forwards providing the wrong parameter map among other things

pei-jung committed:
LPS-99791 These fields are not indexed. No need to reindex.

pei-jung committed:
LPS-99791 Regen

yichenroy committed:
LRCI-566 Auto SF

mtambara committed:
LPS-100349 Make aspectj configurable with a flag

mtambara committed:
LPS-100349 Enable by default on ci

mtambara committed:
LPS-100349 Remove leading indents

mtambara committed:
LPS-100349 Don't have arquillian anymore, don't need this

pyoo47 committed:
LRCI-511 Sort

pyoo47 committed:
LRCI-511 Rename

holatuwol committed:
LPS-100358 Use the portal resource bundle loader when asking for the ResourceBundle for the portal class loader

austinchiang committed:
LRQA-51654 Use comma to confirm email entry

dellawang committed:
update

dellawang committed:
uupdate

JoyceWang08211 committed:
LRQA-51671 Update case to print out keywords parameter

JoyceWang08211 committed:
LRQA-51657 Update macro to declare variable

antonio-ortega committed:
LPS-99515 In case hasOwnerPermission is null we shouldn't return false

dellawang committed:
uuupdate

dellawang committed:
uuuupdate

wincent committed:
LPS-100375 Update liferay-npm-scripts to v8.0.1

Just a maintenance release that ensures that we use the global Prettier
config even when running Prettier from a project directory.

Commit:
LPS-100373 Add missing required dependency

Commit:
LPS-100365 Add missing required dependency

Commit:
LPS-100364 Add missing required dependency

Commit:
LPS-100105 - Replace A.io with fetch

javiergamarra committed:
LPS-100387 launch at least headless tests when changing vulcan, we could launch all API projects...

Commit:
LPS-100107 Standardize Ajax requests in portal-security-service-access-policy-web

p2kmgcl committed:
LPS-100195 SF

Commit:
LPS-100109 Standardize Ajax requests in sync-web

victorg1991 committed:
LPS-100413 Do not rely on sourceTypes for showing label, as they are not defined in content pages

Commit:
LRQA-51404 Add assertions against summary screen details

Commit:
LRQA-51404 Extend test case with assertion against publication summary screen details

p2kmgcl committed:
LPS-99563 Create basic video component

p2kmgcl committed:
LPS-100411 Add missing line

p2kmgcl committed:
LPS-100410 Change 'type=separator' to 'sidebarPanelId=separator' to avoid confusion

p2kmgcl committed:
LPS-100410 Fix typo

p2kmgcl committed:
LPS-100024 Show comments from current experience

Commit:
LPS-98278 SF

Commit:
LPS-98278 Fix unit tests

shuyangzhou committed:
LPS-100358 Flip

brianchandotcom committed:
Record reference to liferay-portal/master-private with git-commit-portal-private as b9d24cc4e0d698653256f6916207a80f572765c9

hhuijser committed:
LPS-99220 Add missing @Deprecated

brianchandotcom committed:
LPS-100059 rename

brianchandotcom committed:
LRCI-511 prep next

dellawang committed:
uuuuupdate
